pages/events/index.vue

```diff
@@ -251,6 +195,34 @@ export default class extends QiskitPage {
 
   &__filters-time {
     margin-top: $layout-03;
+    // deep selector idea taken from
+    // https://vue-loader.vuejs.org/guide/scoped-css.html#deep-selectors
+    ::v-deep .bx--tabs__nav-link {
```

Comment:
Don't do this, @techtolentino, but instead [modify the customized dark theme at `community-theme.scss`](https://github.com/Qiskit/qiskit.org/blob/master/assets/scss/carbon/community-theme.scss).

Let me provide more context: when we started, the last designer thought of giving a dark appearance to all the site so we started to [modify the Carbon theme](https://www.carbondesignsystem.com/guidelines/themes/overview/) for that end.

This is why you can find issues [regarding to how to apply two different themes to the same component](https://github.com/Qiskit/qiskit.org/blob/master/components/learn/TheLearningResourceList.vue#L194-L201).

Since we are moving to the light theme for all the pages, this kind of overriding is not longer needed. What we need to do is to use Carbon themes and customize to make it "white and purple".

Did I make my point?